supervisor/shared/internal_flash.h

```diff
@@ -3,7 +3,7 @@
  *
  * The MIT License (MIT)
  *
- * Copyright 2019 Sony Semiconductor Solutions Corporation
+ * Copyright (c) 2020 Scott Shawcroft, for Adafruit Industries LLC
```

Comment:
should the original copyright be preserved too?  or do you consider it a 100% rewritten file?

Reply:
This is a weird git-ism when trying to detect a moved file. It's empty so changing it is fine.